src/chacha.rs

```diff
@@ -95,6 +111,17 @@ impl ChaChaRng {
     /// users wishing to obtain the conventional ChaCha pseudorandom stream
     /// associated with a particular nonce can call this function with
     /// arguments `0, desired_nonce`.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use rand::{Rng, ChaChaRng};
```

Comment:
`next_u32` isn't a method of `ChaChaRng`, it's part of the `Rng` trait. Compilation fails without it.